src/pages/form-validation/table/index.tsx

```diff
@@ -0,0 +1,222 @@
+    const dispatch = useDispatch();
+
+    const getTable = (): JSX.Element => (
+        <TableContainer component={Paper} className={classes.tableContainer}>
+            <Table>
+                <TableHead style={{ fontWeight: 600 }}>
+                    <TableRow>
+                        <TableCell>ID</TableCell>
+                        <TableCell align="right">Name</TableCell>
+                        <TableCell align="right">Min</TableCell>
+                        <TableCell align="right">Max</TableCell>
+                    </TableRow>
+                </TableHead>
+                <TableBody>
+                    {rows.map((row) => (
+                        <TableRow key={row.id} hover={false}>
+                            <TableCell component="th" scope="row">
+                                {row.id}
+                            </TableCell>
+                            <TableCell align="right">{row.name}</TableCell>
+                            <TableCell align="right">
+                                <TextField
+                                    variant="filled"
+                                    value={row.min}
+                                    classes={{
+                                        root: classes.textFieldRoot,
+                                    }}
+                                    InputProps={{
+                                        classes: {
+                                            input: classes.skinnyInput,
+                                        },
+                                    }}
+                                />
+                            </TableCell>
+                            <TableCell align="right">
+                                <TextField
+                                    variant="filled"
+                                    value={row.max}
+                                    classes={{
+                                        root: classes.textFieldRoot,
+                                    }}
+                                    InputProps={{
+                                        classes: {
+                                            input: classes.skinnyInput,
+                                        },
+                                    }}
+                                />
+                            </TableCell>
+                        </TableRow>
+                    ))}
+                </TableBody>
+            </Table>
+        </TableContainer>
+    );
+
+    const getList = (): JSX.Element => (
+        <>
+            {rows.map((row, index) => (
+                <div key={row.id} style={{ display: 'flex', flexDirection: 'column' }}>
+                    <div style={{ display: 'flex', alignItems: 'center', height: 64 }}>
+                        <Typography
+                            variant={'body1'}
+                            style={{ width: '20%', paddingLeft: theme.spacing(2), color: Colors.gray[500] }}
```

Comment:
in the future we might make this dark theme. Can we do `theme.palette.text.secondary`? Avoid citing `@pxblue/colors` directly.

Reply:
The @pxblue/colors import has been removed for this example.